README.md

```diff
@@ -63,7 +68,37 @@ This file should declare those resources, and any
 customization to apply to them, e.g. _add a common
 label_.
 
-![base image][imageBase]
```

Comment:
Can you please also delete the link definitions here? https://github.com/kubernetes-sigs/kustomize/blob/ee3f506d10d076412a5eeefd3dc8a471a2483413/README.md?plain=1#L169-L170, and the images themselves?

Reply:
@KnVerey, done. Thanks.